srcpkgs/systemd-boot/template

```diff
@@ -49,10 +54,22 @@ case "${XBPS_TARGET_MACHINE}" in
 	*) broken="unknown efi architecture" ;;
 esac
 
-_bins=(ukify)
-_mans=(man/ukify.1 man/systemd-stub.7)
-_efis=("src/boot/efi/linux${_efi_arch}.efi.stub")
-make_build_target="${_bins[*]} ${_mans[*]} ${_efis[*]}"
+_bins=(ukify bootctl)
+_mans=(man/ukify.1 man/systemd-stub.7
+ man/sd-boot.7 man/systemd-boot.7 man/bootctl.1)
+_efis=("src/boot/efi/linux${_efi_arch}.efi.stub"
+ "src/boot/efi/systemd-boot${_efi_arch}.efi")
+_libexecs=(systemd-bless-boot)
+make_build_target="${_bins[*]} ${_mans[*]} ${_efis[*]} ${_libexecs[*]}"
+
+post_patch() {
```

Comment:
can we apply this on all libcs? conditional patches are ugly

Reply:
most of the patches can be applied to all libcs, the main one that's problematic is `musl-misc.patch`. I could pepper it with `#ifdef __GLIBC__`, but it would make the updates a bit more complicated

Reply:
then I'd prefer to make most of them apply universally except the problematic one

Reply:
> then I'd prefer to make most of them apply universally except the problematic one

done